lib/vanity/experiment/ab_test.rb

```diff
@@ -618,8 +622,8 @@ def alternative_for(identity)
       # Saves the assignment of an alternative to a person and performs the
       # necessary housekeeping. Ignores repeat identities and filters using
       # Playground#request_filter.
-      def save_assignment_if_valid_visitor(identity, index, request)
-        return if index == connection.ab_showing(@id, identity) || filter_visitor?(request)
```

Comment:
Hm, just thinking this through - I think this would mean that we start adding bots back into experiment results? (However, they'd be lumped into the default choice.)